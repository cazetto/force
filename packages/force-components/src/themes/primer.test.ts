import primerTheme from './primer';

describe('Primer Theme', () => {
  it('need to match snapshot', () => {
    expect(primerTheme).toMatchSnapshot();
  });
});
