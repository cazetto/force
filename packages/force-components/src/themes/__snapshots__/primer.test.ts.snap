// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primer Theme need to match snapshot 1`] = `
Object {
  "button": Object {
    "colors": Object {
      "contrast": Object {
        "danger": "rgb(255, 255, 255)",
        "disabled": "rgb(255, 255, 255)",
        "primary": "rgb(255, 255, 255)",
        "secondary": "rgb(255, 255, 255)",
        "tertiary": "rgb(255, 255, 255)",
        "warning": "rgb(255, 255, 255)",
      },
      "focusedShadow": Object {
        "danger": "#d73a4980",
        "primary": "#0366d680",
        "secondary": "#28a74580",
        "tertiary": "#6f42c180",
        "warning": "#ffd33d80",
      },
      "hue": Object {
        "danger": "#d73a49",
        "disabled": "#959da5",
        "primary": "#0366d6",
        "secondary": "#28a745",
        "tertiary": "#6f42c1",
        "warning": "#ffd33d",
      },
    },
    "sizes": Object {
      "borderWidth": Object {
        "lg": "2.5px",
        "md": "2px",
        "sm": "1.5px",
        "xl": "3px",
        "xs": "1px",
      },
      "focusedShadow": Object {
        "lg": "3px",
        "md": "3px",
        "sm": "2px",
        "xl": "4px",
        "xs": "2px",
      },
      "font": Object {
        "lg": "15px",
        "md": "13px",
        "sm": "11px",
        "xl": "18px",
        "xs": "10px",
      },
      "padding": Object {
        "lg": "16px 48px",
        "md": "13px 39px",
        "sm": "11px 33px",
        "xl": "18px 54px",
        "xs": "10px 30px",
      },
    },
  },
  "colors": Object {
    "colorBlue": "#0366d6",
    "colorBlue000": "#f1f8ff",
    "colorBlue100": "#dbedff",
    "colorBlue200": "#c8e1ff",
    "colorBlue300": "#79b8ff",
    "colorBlue400": "#2188ff",
    "colorBlue500": "#0366d6",
    "colorBlue600": "#005cc5",
    "colorBlue700": "#044289",
    "colorBlue800": "#032f62",
    "colorBlue900": "#05264c",
    "colorGray": "#6a737d",
    "colorGray000": "#fafbfc",
    "colorGray100": "#f6f8fa",
    "colorGray200": "#e1e4e8",
    "colorGray300": "#d1d5da",
    "colorGray400": "#959da5",
    "colorGray500": "#6a737d",
    "colorGray600": "#586069",
    "colorGray700": "#444d56",
    "colorGray800": "#2f363d",
    "colorGray900": "#24292e",
    "colorGreen": "#28a745",
    "colorGreen000": "#f0fff4",
    "colorGreen100": "#dcffe4",
    "colorGreen200": "#bef5cb",
    "colorGreen300": "#85e89d",
    "colorGreen400": "#34d058",
    "colorGreen500": "#28a745",
    "colorGreen600": "#22863a",
    "colorGreen700": "#176f2c",
    "colorGreen800": "#165c26",
    "colorGreen900": "#144620",
    "colorOrange": "#f66a0a",
    "colorOrange000": "#fff8f2",
    "colorOrange100": "#ffebda",
    "colorOrange200": "#ffd1ac",
    "colorOrange300": "#ffab70",
    "colorOrange400": "#fb8532",
    "colorOrange500": "#f66a0a",
    "colorOrange600": "#e36209",
    "colorOrange700": "#d15704",
    "colorOrange800": "#c24e00",
    "colorOrange900": "#a04100",
    "colorPink": "#ea4aaa",
    "colorPink000": "#ffeef8",
    "colorPink100": "#fedbf0",
    "colorPink200": "#f9b3dd",
    "colorPink300": "#f692ce",
    "colorPink400": "#ec6cb9",
    "colorPink500": "#ea4aaa",
    "colorPink600": "#d03592",
    "colorPink700": "#b93a86",
    "colorPink800": "#99306f",
    "colorPink900": "#6d224f",
    "colorPurple": "#6f42c1",
    "colorPurple000": "#f5f0ff",
    "colorPurple100": "#e6dcfd",
    "colorPurple200": "#d1bcf9",
    "colorPurple300": "#b392f0",
    "colorPurple400": "#8a63d2",
    "colorPurple500": "#6f42c1",
    "colorPurple600": "#5a32a3",
    "colorPurple700": "#4c2889",
    "colorPurple800": "#3a1d6e",
    "colorPurple900": "#29134e",
    "colorRed": "#d73a49",
    "colorRed000": "#ffeef0",
    "colorRed100": "#ffdce0",
    "colorRed200": "#fdaeb7",
    "colorRed300": "#f97583",
    "colorRed400": "#ea4a5a",
    "colorRed500": "#d73a49",
    "colorRed600": "#cb2431",
    "colorRed700": "#b31d28",
    "colorRed800": "#9e1c23",
    "colorRed900": "#86181d",
    "colorYellow": "#ffd33d",
    "colorYellow000": "#fffdef",
    "colorYellow100": "#fffbdd",
    "colorYellow200": "#fff5b1",
    "colorYellow300": "#ffea7f",
    "colorYellow400": "#ffdf5d",
    "colorYellow500": "#ffd33d",
    "colorYellow600": "#f9c513",
    "colorYellow700": "#dbab09",
    "colorYellow800": "#b08800",
    "colorYellow900": "#735c0f",
  },
  "fontSizes": Object {
    "lg": "20px",
    "md": "16px",
    "sm": "14px",
    "xl": "26px",
    "xs": "12px",
  },
  "fontWeights": Object {
    "100": "100",
    "200": "200",
    "300": "300",
    "400": "400",
    "500": "500",
    "600": "600",
    "700": "700",
    "800": "800",
    "900": "900",
  },
  "spaces": Array [
    "0px",
    "4px",
    "8px",
    "16px",
    "32px",
    "64px",
    "128px",
    "256px",
    "512px",
  ],
  "text": Object {
    "colors": Object {
      "danger": "#cb2431",
      "info": "#005cc5",
      "primary": "#05264c",
      "secondary": "#22863a",
      "tertiary": "#4c2889",
      "warning": "#f9c513",
    },
  },
}
`;
